sherpa/astro/xspec/src/_xspec.cc

```diff
@@ -354,6 +360,14 @@ void kyconv_(float* ear, int* ne, float* param, int* ifl, float* photar, float*
 void thcompf_(float* ear, int* ne, float* param, int* ifl, float* photar, float* photer);
 #endif
 
+// XSPEC 12.12.0 changes
+#ifdef XSPEC_12_12_0
+// Note: have dropped the leading 'c_' for this model to follow xsgrbcomp
+void xsgrbjet(const double* energy, int nFlux, const double* params, int spectrumNumber, double* flux, double* fluxError, const char* initStr);
+
+void zxipab_(float* ear, int* ne, float* param, int* ifl, float* photar, float* photer);
```

Comment:
It seems that 'xs' is missing from `zxipab` model name here

Reply:
No it isn't. This is the correct name for this part of the code.

Reply:
ok it is confusing as the model above has 'xs' included and it is called 'xsgrbjet'

Reply:
Yes, that's because we have

```
% egrep 'grbjet|zxipab' ~/local/heasoft-6.29/spectral/manager/model.dat
grbjet         13   1.e-20     1.e+20         c_xsgrbjet   add  0
zxipab       5 0.01         1.e20            zxipab    mul 0 0
```

Note that above we have to say `zxipab_` but `xsgrbjet`, which I can explain but you shouldn't care about.

Welcome to my world...